Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- Fixed an issue where fetching an instance ID wouldn't invoke the callback handler if the instance ID had not changed. (#3251)
```

Comment:
Issue number is preferable to PR when it exists.

Reply:
Mark this is as 4.2.1 since today is code freeze and this will be the one fix for 4.2.1.

Reply:
Done; left the date TBA.